### ISSUE TITLE
Remove DeprecationWarning in floppyforms 1.4?

### ISSUE THREAD

**Issue by EmilStenstrom** [gregmuellegger/django-floppyforms#151]
In https://github.com/gregmuellegger/django-floppyforms/blob/master/floppyforms/models.py#L15 there's a deprecationwarning warning for changes in floppyforms 1.3. Since we're now in 1.4, maybe it's time to remove that warning?

**Comment by melinath** [comment]
It would be good to remove that warning (and make that behavior the default.) This actually looks to be a significant change... according to a note in the `floppyforms.__future__` file, those features require django 1.6+ – so making this change would mean dropping support for two versions of django.

**Comment by melinath** [comment]
@gregmuellegger if that's a thing that we're okay with going ahead on, I'd be happy to make a PR.

**Comment by gregmuellegger** [comment]
Yes, I think we are ok with dropping support for 1.5 and earlier. We could make it so that normal forms work just fine but if you try to subclasses `ModelForm` in 1.5, an exception is raised.

@melinath What's your opinion on this? A PR would be super super awesome. Thanks for stepping up! :beer:
